Memmy-App/memmy
# Language fallback

### PR Creator Checklist

Ensure you've checked the following before submitting your PR:


### Summary

It implements a logic for fallback languages. Ultimately all languages will fall back to `en-US` instead of showing the raw translation key. That way its still visible if a translation is missing but you can still "understand" it (as long as you know english).

It will also now use the full translations code (e.g. en-US, de-DE) if possible but fall back to existing ones (e.g. de-AU will fall back to de-DE)

### Screenshots

Nothing changed in the UI..

### Test Plan

1. Start the Simulator
1. Go to Settings
2. Read the "Check Lemmy Status" label
3. switch to czech or romanian (those don't have that translation
4. It will still read the english label instead of some cryptic stuff
